deploy/k8s/manifest.yaml

```diff
@@ -375,7 +383,35 @@ data:
               - action: update_label
                 label: owner_name
                 new_label: deployment
-
+          - include: k8s.container_fs_reads_total
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's file system read ops
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.reads.rate
+          - include: k8s.container_fs_writes_total
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's file system write ops
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.writes.rate
+          - include: k8s.container_fs_reads_bytes_total
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's file system read bytes
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.reads.bytes.rate
+          - include: k8s.container_fs_writes_bytes_total
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's file system written bytes
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.writes.bytes.rate
+          - include: k8s.container_fs_usage_bytes
+            action: insert
+            match_type: regexp
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's file system usage in bytes
+            new_name: k8s.pod.fs.usage.bytes
```

Comment:
you should be also summing this for each pod. Check how @michalJakubis is doing that in his PR: https://github.com/solarwinds/swi-k8s-opentelemetry-collector/pull/97/files